specs/v1.yaml

```diff
@@ -1257,6 +1394,93 @@ components:
           example: direct
         address:
           $ref: "#/components/schemas/GrantAddress"
+    RecurringGrant:
+      type: object
+      description: |-
+        A RecurringGrant represents a successfully initiated recurring grant request with a Donor Advised Fund. 
+        RecurringGrants are created when a person interacts with an instance of Chariot Connect and successfully submits a recurring grant and completes the workflow. 
+        On creation, each RecurringGrant object will also have a Grant object created to represent the first grant of the recurring grant.
+        There can be many recurring grants associated with a Connect object and therefore a Nonprofit.
+      required:
+        - id
+        - workflowSessionId
+        - fundId
+        - amount
+      properties:
+        id:
+          type: string
+          readOnly: true
+          description: The unique identifier for the object
+          format: uuid
+          example: cfe09e64-6a74-4dab-a565-361185a6f248
+        trackingId:
+          type: string
+          readOnly: true
+          description: The tracking ID for the grant
+          example: L9E182VBGP
+        workflowSessionId:
+          type: string
+          readOnly: true
+          description: ID of the Connect Workflow Session associated with this grant
+          format: uuid
+          example: 2d4b2a43-a5b4-4be1-ad1f-f932016ca4a6
+        fundId:
+          type: string
+          readOnly: true
+          description: ID of the donor advised fund
+          example: daf-id
+        externalGrantId:
+          type: string
+          description: ID of the grant associated with the donor advised fund
+          example: 897823sdjf8sfjs
+        createdAt:
+          type: string
+          readOnly: true
+          format: date-time
+          description: Time when this object was created; expressed in ISO 8601 format
+          example: "2020-07-10 15:00:00.000"
+        updatedAt:
+          type: string
+          readOnly: true
+          format: date-time
+          description: Time when this object was last updated; expressed in ISO 8601 format
+          example: "2021-07-11 15:34:00.000"
+        amount:
+          type: number
+          format: integer
+          description: The grant amount expressed in units of whole cents
+          example: 15000
+        status:
```

Comment:
i think we should remove status from recurring grants for now

Reply:
I thought that I removed this. Thanks for catching. I also removed the metadata below.

Reply:
let's keep metadata for now - that's useful for clients to pass in arbitrary key/value pairs that they want to attach to the objects that are stored in chariot's system